docs/pages/contributors.rst

```diff
@@ -14,28 +14,42 @@ counts, so do not hesitate!
 2. Fork `the repository <https://github.com/tmbo/questionary>`_ on GitHub to
    start making your changes.
 
-3. Write some tests that show the bug is fixed or that the feature works as
+3. Configure development environment.
+
+  a. create python virtual environment, eg
+
+  .. code-block:: console
+
+    virtualenv .env && source .env/bin/activate
```

Comment:
This should not be necessary when using poetry.

Reply:
correct, will remove